applications/server/controllers/synchronization.py

```diff
@@ -19,7 +15,13 @@ def receive_logs():
                         {"modulename":"my_module","log_level":2,"log_message":"some other message","time_stamp":"2016-03-19 20:49:41"}
                     ]
             }
-    """
+
+   :return: Dictionary containing whether the request posted is a success or not and the latest synchronization time_stamp
+   :rtype: Dictionary
+"""
+
+@request.restful()
+def receive_logs():
     def GET(*args, **vars):
         logs = db(db.logs).select(orderby="received_time_stamp DESC",limitby=(0,3))
         return response.json(logs)
```

Comment:
You would like to write: 

return json_plus.Serializable.dumps(myobject)

but this may not set the content-type headers. 
You can also do: 
return response.json_plus.Serializable.dumps(....)

but this is not implemented.